example ghul.template build fails

peter@peter-ThinkPad-X220:~/ghul$ dotnet new --install ghul.templates
The following template packages will be installed:
   ghul.templates

ghul.templates is already installed, version: 0.2.1, it will be replaced with version .
ghul.templates::0.2.1 was successfully uninstalled.
Success: ghul.templates::0.2.1 installed the following templates:
Template Name               Short Name     Language  Tags   
--------------------------  -------------  --------  -------
ghūl class library package  ghul-classlib  ghūl      Console
ghūl console application    ghul-console   ghūl      Console

peter@peter-ThinkPad-X220:~/ghul$ mkdir my-console-project
peter@peter-ThinkPad-X220:~/ghul$ cd my-console-project
peter@peter-ThinkPad-X220:~/ghul/my-console-project$ dotnet new ghul-console
The template "ghūl console application" was created successfully.
peter@peter-ThinkPad-X220:~/ghul/my-console-project$ dotnet run
/home/peter/.nuget/packages/ghul.targets/1.0.0/build/ghul.targets.targets(71,5): error MSB3073: The command "dotnet ghul-compiler @.build.rsp" exited with code 1. [/home/peter/ghul/my-console-project/my-console-project.ghulproj]

The build failed. Fix the build errors and run again.

FYI - environment
peter@peter-ThinkPad-X220:~/ghul/my-console-project$ dotnet --list-sdks
5.0.101 [/home/peter/dotnet/sdk]
6.0.100 [/home/peter/dotnet/sdk]

NAME="Ubuntu"
VERSION="20.04.3 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.3 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal

peter@peter-ThinkPad-X220:~/ghul/my-console-project$ dotnet tool restore
Tool 'ghul.compiler' (version '0.5.0') was restored. Available commands: ghul-compiler

Restore was successful.
peter@peter-ThinkPad-X220:~/ghul/my-console-project$ dotnet run
/home/peter/.nuget/packages/ghul.targets/1.0.0/build/ghul.targets.targets(71,5): error MSB3073: The command "dotnet ghul-compiler @.build.rsp" exited with code 1. [/home/peter/ghul/my-console-project/my-console-project.ghulproj]

The build failed. Fix the build errors and run again.

[bin.zip](https://github.com/degory/ghul/files/7794225/bin.zip)

peter@peter-ThinkPad-X220:~/ghul$ dotnet nuget locals --clear all
Clearing NuGet HTTP cache: /home/peter/.local/share/NuGet/v3-cache
Clearing NuGet global packages folder: /home/peter/.nuget/packages/
Clearing NuGet Temp cache: /tmp/NuGetScratch
Clearing NuGet plugins cache: /home/peter/.local/share/NuGet/plugins-cache
Local resources cleared.
peter@peter-ThinkPad-X220:~/ghul$ mkdir test-project
peter@peter-ThinkPad-X220:~/ghul$ cd test-project
peter@peter-ThinkPad-X220:~/ghul/test-project$ dotnet new ghul-console
The template "ghūl console application" was created successfully.
peter@peter-ThinkPad-X220:~/ghul/test-project$ dotnet tool restore
Tool 'ghul.compiler' (version '0.5.0') was restored. Available commands: ghul-compiler

Restore was successful.
peter@peter-ThinkPad-X220:~/ghul/test-project$ dotnet build /v:q >output.txt 2>&1
peter@peter-ThinkPad-X220:~/ghul/test-project$ dotnet run
/home/peter/.nuget/packages/ghul.targets/1.0.0/build/ghul.targets.targets(71,5): error MSB3073: The command "dotnet ghul-compiler @.build.rsp" exited with code 1. [/home/peter/ghul/test-project/test-project.ghulproj]

The build failed. Fix the build errors and run again.



[output.txt](https://github.com/degory/ghul/files/7794581/output.txt)

--- a/.config/dotnet-tools.json
+++ b/.config/dotnet-tools.json
@@ -15,7 +15,7 @@
       ]
     },
     "ghul.compiler": {
-      "version": "0.4.1",
+      "version": "0.5.0",
       "commands": [
         "ghul-compiler"
       ]

--- a/.vscode/extensions.json
+++ b/.vscode/extensions.json
@@ -1,0 +1,5 @@
+{
+    "recommendations": [
+        "degory.ghul"
+    ]
+}

--- a/src/driver/main.ghul
+++ b/src/driver/main.ghul
@@ -383,7 +383,7 @@ namespace Driver is
             fi
             
             let runtime_config = File.create_text(runtime_config_file);
-            runtime_config.write("{\"runtimeOptions\":{\"tfm\":\"netcoreapp6.0\",\"framework\":{\"name\":\"Microsoft.NETCore.App\",\"version\":\"" + IoC.CONTAINER.instance.assemblies.sdk_version + "\"}}}");
+            runtime_config.write("{\"runtimeOptions\":{\"tfm\":\"netcoreapp6.0\",\"framework\":{\"name\":\"Microsoft.NETCore.App\",\"version\":\"6.0.0\"}}}");
             runtime_config.close();
 
             if !RuntimeInformation.is_o_s_platform(OSPlatform.windows) then

